### PR TITLE
call list subheaders

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
<s>WIP; testing and finishing depends on #4323 (arrays being proper arrays)</s>

testable

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
ping, #4323 is merged

**Comment by MaximilianKrambach** [comment]
@emanuelschuetze done

File: client/src/app/site/motions/services/motion-pdf.service.ts [hunk 1]
```diff
@@ -547,6 +574,7 @@ export class MotionPdfService {
                 }
             }
         };
+        console.log(table);
```
Comment (emanuelschuetze) [inline client/src/app/site/motions/services/motion-pdf.service.ts]: Please remove this line

**Comment by MaximilianKrambach** [comment]
fixed and rebased @emanuelschuetze

**Pull request merged**